src/babel.js

```diff
@@ -23,4 +26,10 @@ module.exports = babelJest.default.createTransformer({
 	],
 	babelrc: false,
 	configFile: false,
-});
+	overrides: config.files.size ? [config.options] : [],
+};
+
+module.exports = {
+	...babelJest.default,
+	createTransformer: () => babelJest.default.createTransformer(options),
+};
```

Comment:
What's the reason behind this change?

Reply:
That was accidentally left over from earlier attempts. Please check the latest.